.fixtures.yml

```diff
@@ -26,7 +26,9 @@ fixtures:
     selinux_core:
       repo: "https://github.com/puppetlabs/puppetlabs-selinux_core"
       puppet_version: ">= 6.0.0"
-    systemd:       "https://github.com/camptocamp/puppet-systemd.git"
+    systemd:
+      repo: 'https://github.com/camptocamp/puppet-systemd'
+      ref: '2.12.0' 
```

Comment:
It'd be great if you could open a PR with just this change in it, I am needing locally for a few PRs myself.

Reply:
Done at https://github.com/theforeman/puppet-foreman_proxy_content/pull/336

Reply:
Thanks!